### PR TITLE
Bump io lst

### PR DESCRIPTION
**Pull request by rlopezcoto**

### CODE REVIEW

**Comment by morcuended** [comment]
Test failing because I'll take a while until ctapipe_io_lst v0.22.2 is available in conda forge

**Comment by morcuended** [comment]
@maxnoe or @moralejo can you please trigger the conda release in https://github.com/conda-forge/ctapipe_io_lst-feedstock if not done yet?

File: environment.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ dependencies:
   - pandas>=2
   - pymongo
   - seaborn
-  - ctapipe_io_lst=0.22
+  - ctapipe_io_lst=0.22.2
```
Comment (maxnoe) [inline environment.yml:34]: This will prevent newer 0.22.x versions from being installed once released.

Comment (maxnoe) [reply environment.yml:34]: =0.22 will install the newest 0.22.x version

**Comment by rlopezcoto** [comment]
done in #1779

**Pull request closed without merging**